source/base/data_out_base.cc

```diff
@@ -1584,6 +1584,31 @@ namespace DataOutBase
 
 
 
+  template <int dim, int spacedim>
+  void
+  Patch<dim,spacedim>::swap (Patch<dim,spacedim> &other_patch)
+  {
+    std::swap (vertices, other_patch.vertices);
+    std::swap (neighbors, other_patch.neighbors);
+    std::swap (patch_index, other_patch.patch_index);
+    std::swap (n_subdivisions, other_patch.n_subdivisions);
+    data.swap (other_patch.data);
+    std::swap (points_are_available, other_patch.points_are_available);
+
+//      Point<spacedim> vertices[GeometryInfo<dim>::vertices_per_cell];
+//      unsigned int neighbors[dim > 0
+//                             ?
+//                             GeometryInfo<dim>::faces_per_cell
+//                             :
+//                             1];
+//      unsigned int patch_index;
+//      unsigned int n_subdivisions;
+//      Table<2,float> data;
+//      bool points_are_available;
+  }
```

Comment:
Yes, complete artifact of how I wrote this function. Absolutely no good reason to leave it in. Absolutely no good reason in fact why I should submit a patch that still has this in there :-(

In any case, I've fixed it.